### PR TITLE
Mobile menu

### PR DESCRIPTION
**Pull request by Roland-Ntwali**
In this PR:

I made the menu button open and close.

### CODE REVIEW

File: menu.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+const menu_button = document.getElementById("menu_btn");
+const close_menu = document.getElementById("close_menu");
+
+menu_button.addEventListener("click", () => {
+  mobile_menu.classList.add("show");
+});
+
+close_menu.addEventListener("click", () => {
+  mobile_menu.classList.remove("show");
+});
```
Comment (brainconnect93) [inline menu.js]: - [ ] Hi @Roland-Ntwali, having made a great effort working on the mobile menu aspect of your portfolio project, I noticed there are still some things you need to make changes on. 
   - According to what we have in the Figma design after the mobile menu is displayed the content underneath it is `blur` but in your own design the contents are not `blur`. You can achieve blurring of the background content by making use of the CSS property called `backdrop-filter: blur(5px);`
   - Secondly, according to the project requirement when a user clicks on any of the mobile menu links it should direct me to the corresponding page in your project and the mobile menu should disappear, I noticed when I clicked the link, the mobile menu is not disappearing. Kindly fix this
   > N.B:  req

Figma design 👇 

real-mob

Your design 👇 

rola

**Review by brainconnect93 — CHANGES_REQUESTED** [review]
## STATUS: Changes Required :recycle:

### Hi @Team, 👋 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### To Highlight :1st_place_medal: 
-  Has a proper Pull Request. ✔️  
- No Linter errors :heavy_check_mark: 
- Good use of Github flow ✔️ 

### Changes Required

- [x]  _Please check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @brainconnect93 ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 2]
```diff
@@ -60,11 +60,47 @@ body {
...
+#mobile_menu {
+  position: fixed;
+  display: none;
+  flex-direction: column;
+  justify-content: start;
+  background-color: #6070ff;
+  gap: 8px;
+  font-size: larger;
+  list-style-type: none;
+  opacity: 0.9;
```
Comment (huseyinbiyik) [inline style.css]: Good job with the links, now nav is closing automatically 👌 But let's add the cool backdrop blur as the previous reviewer requested.
First, please remove the opacity: 0.9. 
Now change your background HEX color value to RGBA. It is going to be `background-color: rgba(96, 112, 255, 0.9);`
Now it is going to blurred. 👍

**Review by huseyinbiyik — CHANGES_REQUESTED** [review]
Hi team,

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
tenor

 ## Highlights
- Mobile nav functioning properly. 👌
- Github flow followed 👌


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- Unfortunately, the previous reviewer missed one requirement about linters. Kindly add  new linter setup for this project as mentioned on project requirements.
Screenshot 2023-01-31 at 10 11 25 PM

- You can fix your header to the top to increase the UX of your website like [here](https://www.figma.com/proto/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39:122&scaling=min-zoom&page-id=0:1). 😎

- I think it would be better if you can remove unrelated files in your repository. You have test.md file in your repository. To keep your repository professional, removing it would be a good choice 👍

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._∂

File: style.css [hunk 3]
```diff
@@ -60,11 +67,46 @@ body {
...
+  font-size: larger;
+  list-style-type: none;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  margin: 0;
+  z-index: 99;
+  backdrop-filter: blur(5px);
+}
```
Comment (jerryowusu) [inline style.css:91]: [OPTIONAL] Consider using `mix-blend-mode: multiply` for the overlay blur effect since that gives you the best view. :wink:

**Review by jerryowusu — APPROVED** [review]
Hi @Roland-Ntwali , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

good job


## Highlights
No linter errors :+1: 
Great effort in making the requested changes :+1: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**